opentracing-metrics-prometheus-spring-autoconfigure/pom.xml

```diff
@@ -38,6 +38,12 @@
     </dependency>
 
     <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-spring-web-autoconfigure</artifactId>
+      <scope>provided</scope>
```

Comment:
I am not sure whether this should be provided. In different projects I have seen that starters depended on starters.